src/OrchardCore.Modules/OrchardCore.Demo/Startup.cs

```diff
@@ -64,6 +64,9 @@ public override void Configure(IApplicationBuilder builder, IEndpointRouteBuilde
 
             var demoAdminControllerName = typeof(AdminController).ControllerName();
 
+            // While you can define admin routes like this, we suggest adding the [Admin("path after the admin prefix")]
```

Comment:
@Piedone this is a demo module not a training module. This would be a great add to the training module and the docs for reference. Not sure we should have it as part of the framework. But, if you still think we should keep it, I am good with it.

Reply:
Why would a comment hurt here though? It demonstrates two approaches; it's useful to have an explanation of why there are two.